modules/filter_fastq_modules.py

```diff
@@ -0,0 +1,45 @@
+# подсчёт GC-состава рида.
+def calculate_gc_bounds(seq):
+    """
+    Выполняет подсчёт Q-score по формуле Unicode-значение символа - 33
+
+    Parameters
+    ----------
+    seq : str
+        последовательность нуклеотидов
+
+    Returns
+    -------
+    gc_content : float
+        Выводит значение содержание "G" и "C" в последовательности
+    """
+    count_g = seq.lower().count("g")
+    count_c = seq.lower().count("c")
+    gc_content = ((count_g + count_c) / len(seq)) * 100
+    return gc_content
+
+
+# подсчёт качества рида.
+def calculate_quality_threshold(quality):
+    """
+    Выполняет подсчёт Q-score по формуле Unicode-значение символа - 33
+
+    Parameters
+    ----------
+    quality : str
+        строка из символов и букв, характеризующих
+        качество прочтения нуклеотидов
+
+    Returns
+    -------
+    q_score : float
+        Выводит значение среднего Q-score
+    """
+    seq_length = len(quality)
+    summ_q_score = 0
+    for letter in quality:
+        letter_q_score = ord(letter) - 33
+        summ_q_score += letter_q_score
+
+    q_score = summ_q_score / seq_length
+    return q_score
```

Comment:
```suggestion
    return sum(ord(letter) - 33 for letter in quality) / len(quality) 
```
(хотя предположу что вы и так знаете о такой возможности, и если ваш текущий вариант вам кажется более красивым, то можно его и оставить)

Reply:
количество действий можно сократить и правда. обычно избегаю однострочники во имя понятности кода, но тут он вполне.